website/blog/2023-02-02-5-database-alternatives-mongodb-2023.md

```diff
@@ -0,0 +1,121 @@
+---
+slug: 5-database-alternatives-mongodb-2023
+title: "MongoDB Alternatives: 5 Database Alternatives to MongoDB for 2023"
+author: Alexander Fashakin
+image: /img/blog/mongodb-alternatives.png
+description: "The top 5 MongoDB-compatible alternatives to MongoDB include: FerretDB, DocumentDB, CosmosDB, GaussDB(for Mongo), and ToroDB."
```

Comment:
```suggestion
description: "The top 5 MongoDB-compatible alternatives to MongoDB include: FerretDB, DocumentDB, CosmosDB, GaussDB (for Mongo), and ToroDB."
```

Reply:
It's not a mistake 😁. That's how the company writes the name of the product.

https://www.huaweicloud.com/intl/en-us/product/gaussdbformongo.html

Reply:
:sweat_smile: